hls4ml/model/layers.py

```diff
@@ -111,17 +112,18 @@ def __init__(self, model, name, attributes, inputs, outputs=None, initialize=Tru
             layer_config = self.model.config.get_layer_config(self)
             for config_key, config_value in layer_config.items():
                 config_key = convert_to_snake_case(config_key)
-                if config_key in self.attributes:
```

Comment:
I am worried about removing this warning. It basically indicates that the attribute you copied is overwritten, so you may not be getting what you expect.